target/linux/ath79/nand/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -3,13 +3,16 @@
 [ -e /lib/firmware/$FIRMWARE ] && exit 0
 
 . /lib/functions/caldata.sh
-. /lib/functions/k2t.sh
 
 board=$(board_name)
 
 case "$FIRMWARE" in
 "ath10k/cal-pci-0000:00:00.0.bin")
 	case $board in
+	domywifi,dw33d)
+		caldata_extract "art" 0x5000 0x844
+		ath10kcal_patch_mac $(mtd_get_mac_binary art 0x12)
```

Comment:
-> ath10k_patch_mac

Reply:
Ah, so this could really be 0x12 in this case. Please check whether there is a valid MAC address in art 0x5006.
I would be interested to see a full hexdump of the art partition. Refer to https://openwrt.org/docs/guide-developer/mac.address for further instructions (if necessary).

Reply:
‘mtd_get_mac_binary art 0x12’Can return the correct MAC address,

Reply:
Okay, still please provide us with an overview of the MAC addresses assignment on stock firmware and the art location. This could be added to the commit message later. What about 0x1002 and 0x5006?

Reply:
eth0 gets the mac from the art address 0x0, eth1 is 0x6, 0xc is the mac address of the ath9k device, and 0x12 is the ath10k device. These mac addresses are identical to the oem firmware.
0x1002 and 0x5006 are not valid mac addresses.
Please forgive me for not knowing English.

Reply:
Okay, perfect. Please add this information to the commit message, e.g.
MAC address overview:
eth0: 0x0
eth1: 0x6
ath9k: 0xc
ath10k: 0x12
No valid addresses in 0x1002 and 0x5006. All addresses match the OEM firmware.
```

Reply:
Don't forget to change the function name.